### DISCUSSION TITLE
Sonoff POW Elite 16A (POWR316D) cannot be flashed

### DISCUSSION THREAD

**Discussion by MecMax** [arendst/Tasmota#20821] [Support request]
Hello!
I bought 2 Sonoff POW Elite 16a. But flashing on Tasmota is not possible!
Following situation:
Windows 11 PC
AZ Delivery FT232RL USB - TTL adapter
Sonoff POW Elite 16a
Software: Driver was installed automatically
I have now tested the following flash-software (without any success):
NodeMCU PyFlasher
ESPHome-Flasher-1.4.0-Windows-x64
Tasmota web installer

process:
Adapter and Sonoff connected correctly (TX and RX crossed)
Pressed the on/off button on the Sonoff and then plugged in the USB
USB port is suggested
Tasmota32.bin selected
Flashing process started
Report:
Using 'COM3' as serial port.
Connecting......................................
Unexpected error: ESP Chip Auto-Detection failed: Failed to connect to Espressif device: No serial data received.
For troubleshooting steps visit: https://github.com/espressif/esptool#troubleshooting
Using 'COM3' as serial port.
Unexpected error: could not open port 'COM3': PermissionError(13, 'Zugriff verweigert', None, 5)

I'm completely stumped. Does anyone have an idea?

**Comment by sfromis** [comment]
Better use recommended tools from the start, instead of just picking something remotely similar. And beware that those "FTDI" fakes are unsuitable for flashing, due to not delivering enough power. Again, use what's recommended; at least make sure to have at least 150 mA of 3.3V power.
https://tasmota.github.io/docs/Getting-Started/

**Reply by MecMax** [reply]
Hello sfromis!
Your tip helped me. I bought a recommended adapter and a connection and transfer was possible immediately. Unfortunately I was misled by a video.
So thank you again!

**Reply by MecMax** [reply]
Hello! One more question.
I followed the BangerTech video. I downloaded tasmota32.bin (12.0.2.3) from there and flashed it. Everything worked. Now I'm trying this with the current version from the Tasmota site. I tried it with
tasmota32.bin
tasmota32-de.bin
tasmota32.factory.bin
In none of these versions does the device connect via WiFi. what am I doing wrong?

**Reply by sfromis** [reply]
Don't expect it to work fully until you disconnect the wiring for flashing, and instead power via mains. And the `.factory` variants are specifically and only for wired flashing.

**Reply by MecMax** [reply]
Hello! After flashing, I allways disconnected the cables and tested with the normal power supply. The device does not connect to the WLAN.
In the meantime I also tested the upgrade. Device does not upgrade even though no error message appears.
Disconnects briefly and is then available again with the original version.

**Reply by MecMax** [reply]
Hello! I was able to solve the problem myself. Did the whole process again and used the web installer. Now works as it should with the current Tasmota.
Thanks for the support!
